app/style.css

```diff
@@ -15,7 +15,7 @@ body {
 }
 #top {
     display: flex;
-    justify-content: space-between;
+    justify-content: flex-start;
```

Comment:
Okay. Didn't know about wrapping a div. If I change this, do I make a new commit?